index.html

```diff
@@ -4,7 +4,7 @@
     <meta charset="UTF-8" />
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Quiz App</title>
+    <title class="title">Quiz App!</title>
```

Comment:
You don't need to apply classes to elements within `head` since you can not apply styles to them. :) 
I guess you wanted to give your `h1` the class, right?